web/src/input_pill.ts

```diff
@@ -364,6 +370,10 @@ export function create<T>(opts: InputPillCreateOptions<T>): InputPillContainer<T
 
                 return;
             }
+
+            if (store.onKeyDownInputFunction !== undefined) {
+                store.onKeyDownInputFunction();
+            }
```

Comment:
Let's call this `onInputHook`; I think that's clear enough.

Also you can probably spell this `store.onInputHook?()`, without the if statement, since it's TypeScript.